7.2.0/h2/shared_db_cleanup.sh

```
@@ -0,0 +1,128 @@
+      elif [[ "$table" == UM_* ]]; then
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE UM_TENANT_ID = $TENANT_ID;"
+      elif [[ "$table" == REG_* ]]; then
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE REG_TENANT_ID = $TENANT_ID;"
+      fi
+    done
+    DELETE_PROCEDURE+="COMMIT;"
+
+
+    # Execute the delete procedure
+    echo "Executing deletion procedure for TENANT_ID=$TENANT_ID..."
+    echo "$DELETE_PROCEDURE" | java -cp "$H2_JAR_PATH" org.h2.tools.Shell -url "$DB_URL" -user "$DB_USER" -password "$DB_PASSWORD" 2>&1 > /tmp/sql_output.log
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)